geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/parameters/Parameter.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.redis.internal.parameters;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+import org.apache.geode.redis.internal.RedisCommandType;
+import org.apache.geode.redis.internal.netty.Command;
+import org.apache.geode.redis.internal.netty.ExecutionHandlerContext;
+
+public class Parameter {
+
+  private int arity;
+  private List<RedisCommandType.Flag> flags = new ArrayList<>();
+  private int firstKey = 1;
+  private int lastKey = 1;
+  private int step = 1;
+  private List<BiConsumer<Command, ExecutionHandlerContext>> predicates = new ArrayList<>();
```

Comment:
None of the predicates use an `ExecutionHandlerContext`, so this could be changed to be just a `Consumer<Command>`.

Reply:
Yes, good observation! That cleans things up nicely.